### DISCUSSION TITLE
How can I continue searching even if there are no occurrences?

### DISCUSSION THREAD

**Discussion by wochap** [folke/flash.nvim#369] [Q&A] [answered]
I want to be able to manually close require('flash').jump(). Sometimes I type too quickly and make typos, causing the flash to finish and all my key inputs to affect my current buffer
I would also like to receive notifications when Flash cannot find anything, like `/` does

If it's not possible, could you direct me to where I might modify the code in flash.nvim to make it possible?

**Comment by alex35mil** [comment]
+1. I would also prefer to avoid auto-bail on no-match for the mentioned reason.

**Comment by AllergicMushroom** [comment, marked as answer]
I was also looking for this feature, and although it's not documented, I've found a way to do it :
`require('flash').jump({search={mode='search'}}))`.

I'm not sure how that links to the more general `mode='search'` option. Below there are two examples of configurations. The first doesn't do what we want, while the second does.
```
return {
    'folke/flash.nvim',
    event = 'VeryLazy',
    keys = {
        { '<leader>ff', function() require("flash").jump() end, mode = 'n', desc = "Flash" },
    },
    config = {
        mode = 'search',
    }
}
```

```
return {
    'folke/flash.nvim',
    event = 'VeryLazy',
    keys = {
        { '<leader>ff', function() require("flash").jump({ search = { mode = 'search' } }) end, mode = 'n', desc = "Flash" },
    },
    config = {
    }
}
```

I'd still prefer I could use fuzzy finding while also not stopping the search abruptly if no match is found, but this is already a great improvement.
